src/StackExchange.Redis/RedisDatabase.cs

```diff
@@ -1580,6 +1580,30 @@ public Task<long> SortedSetAddAsync(RedisKey key, SortedSetEntry[] values, When
             return ExecuteAsync(msg, ResultProcessor.Int64);
         }
 
+        public RedisValue[] SortedSetCombine(SetOperation operation, RedisKey[] keys, double[]? weights = null, Aggregate aggregate = Aggregate.Sum, CommandFlags flags = CommandFlags.None)
+        {
+            var msg = GetSortedSetCombineCommandMessage(operation, keys, weights, aggregate, false, flags);
```

Comment:
On these, let's make the code a bit more readable for consumers by labelling the `false` arg please:
```suggestion
            var msg = GetSortedSetCombineCommandMessage(operation, keys, weights, aggregate, withScores: false, flags);
```